doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc

```diff
@@ -101,6 +99,8 @@ This is because the first `Prepare hardware spec` task has been completed with t
 . Go to *Menu* -> *Manage* -> *Process Instances* and open the *Order for IT hardware* process instance.
 . Open the *Diagram* tab. The `Order rejected` and `Place order` processes are now marked as *Completed*.
 +
+*<MH>Dev, I am unable to replicate this diagram with the milestone highlighted in Stunner. Can you provide me with a new version in Stunner?</MH>*
```

Comment:
Do we need to document steps 17 to 21? If this section is to show how you can use rules in a case, wouldn't the result of step 16 show this sufficiently?  So my suggestion would be to remove steps 17 - 21 here too.

Reply:
Agreed. Removed steps and image.